main/zoe/guide/contracts/barter-exchange.md

```diff
@@ -0,0 +1,17 @@
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/barter-exchange.js) (Last updated: 6/30/2020)
```

Comment:
Did you intend to link to a particular version of the code?

Reply:
We have this format for the rest of our contract docs - two links. One is a permalink and one is not. I'm not sure I like it anymore, but this is consistent.

Reply:
No, bad copy on my part, will fix.